src/index.ts

```diff
@@ -1,15 +1,33 @@
 export { default as Button } from './Button/Button';
+
 export { default as Checkbox } from './Checkbox/Checkbox';
+
 export { default as Collapsible } from './Collapsible/Collapsible';
+
+export { default as Filter } from './Filter';
+export * from './Filter'
```

Comment:
Is this export * from Filter intentional?

Reply:
@michaelpuzon - Yes, though it just changed again slightly. After some discussion with @JonTheBurley, we think it makes sense to export the util functions, types, and filter modules from the root. We need a way to expose those, but don't see an immediate need to scope them further (which is what I was originally attempting to do). Let me know if that needs further clarification.

Reply:
With the most recent change, imports will work as follows:
`import { Filter, useFilter, FILTER_MODE_OPTIONS, getApiPostBody, getDefaultValue } from '@toyota-research-institute/lakefront';`

Reply:
I think that looks good. If we decide we need them separated later, we can work on that. I don't think it matters too much to the user how it looks when it's imported at the moment.

Reply:
My argument against this was finding it harder to debug later on. 
Let's you had this statement `import { getDefaultValue } from '@toyota-research-institute/lakefront';`
It's not that obvious that this was in the Filter part of lakefront. 
The other point is that it might be easy to have duplicate hooks and functions exported. An example would be having a future `getDefaultValue` function in a future module in lakefront.
I don't have a better solution right now. If you can think of a better solution down the road, feel free to revisit this.